plugins/ag-grid/src/js/src/components/LoadingOverlay.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import classNames from 'classnames';
+import LoadingSpinner from './LoadingSpinner';
+
+type LoadingOverlayProps = {
+  className?: string | null;
+  'data-testid'?: string;
+};
+
+/**
+ * A loading overlay that handles displaying a loading spinner or an error message
```

Comment:
Error message doesn't seem supported
```suggestion
 * A loading overlay that handles displaying a loading spinner
```